### DISCUSSION TITLE
Relationship select field not submitting on creating record but working on editing record

### DISCUSSION THREAD

**Discussion by naamsachin** [filamentphp/filament#8781] [Help]
Relationship select field not submitting on creating record but working on editing record

**Comment by ArtrenH** [comment]
I think, I got the same problem. Have you solved it somehow?

**Comment by ArtrenH** [comment]
nvm, found your other thread: https://github.com/filamentphp/filament/issues/8789
Maybe you could close this and link to the other one.
